Clover/app/src/main/java/org/floens/chan/ui/view/MultiImageView.java

```diff
@@ -507,6 +526,28 @@ private void cleanupVideo(PlayerView videoView) {
         videoView.getPlayer().release();
     }
 
+    public void toggleTransparency() {
+        CustomScaleImageView imageView = findScaleImageView();
+        GifImageView gifView = findGifImageView();
+        if (imageView == null && gifView == null) return;
+        boolean isImage = imageView != null && gifView == null;
+        if (backgroundToggle) {
```

Comment:
you might be able to remove this exterior conditional by doing a ternary first
```
backgroundColor = backgroundToggle ? Color.TRANSPARENT : Color.BACKGROUND_COLOR
if (isImage) {
    imageView.setTileBackgroundColor(backgroundColor);
} else {
    gifView.getDrawable().setColorFilter(backgroundColor, PorterDuff.Mode.DST_OVER);
}
backgroundToggle = !backgroundToggle
```

Reply:
Yep, makes sense to me.